### PR TITLE
Initial Codeowners file

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes  RNBW-3002

## What changed (plus any additional context for devs)
- Added CODEOWNERS file

## PoW (screenshots / screen recordings)
none

## Dev checklist for QA: what to test
no need to test anything

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+src/styled-thing @terrysahaidak @osdnk  
+
+# styles
+src/styled-thing @terrysahaidak @osdnk @markdalgleish
+
+# utils
+src/utils @brunobar79 @jinchung
+
+# App
+src/App.ts @brunobar79 @jinchung
```
Comment (osdnk) [inline CODEOWNERS:139]: this is still `App.js` unfortunately. I think you can count me in here, I did pretty a lot here

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+.watchmanconfig @brunobar79 @osdnk
+# SECURITY / APP INFRA
+
+.yarnrc @brunobar79
+
+# APP INFRA
+app.json @jinchung @brunobar79
+
+# SECURITY / APP INFRA
+audit-ci.json @brunobar79 @skylarbarrera
```
Comment (tchayen) [inline CODEOWNERS:166]: I am often fixing CI audits and editing this file from time to time. I'd add myself here.

Comment (tchayen) [inline CODEOWNERS:166]:
```suggestion
audit-ci.json @brunobar79 @skylarbarrera @tchayen
```

I am often fixing CI audits and editing this file from time to time. I'd add myself here.

File: CODEOWNERS (hunk 1 above)
Comment (brunobar79) [reply CODEOWNERS:139]:
```suggestion
src/App.js @brunobar79 @jinchung @osdnk 
```

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+src/raps/registerENS.ts @estebanmino @jxom
+
+# charts, shadow stack, cool modals, sheets
+src/react-native-animated-charts @osdnk @terrysahaidak
+src/react-native-cool-modals @osdnk @terrysahaidak
+src/react-native-shadow-stack @osdnk @terrysahaidak
+src/react-native-yet-another-bottom-sheet @osdnk @terrysahaidak
+
+# react query
+src/react-query @jinchung @derHowie
```
Comment (jxom) [inline CODEOWNERS:108]: am happy to be added here (from a react-query API perspective) as I did create this folder 😛

Comment (jxom) [inline CODEOWNERS:108]:
```suggestion
src/react-query @jinchung @derHowie @jxom
```

File: CODEOWNERS (hunk 2 above)
Comment (jinchung) [reply CODEOWNERS:166]: hey! codeowners file just means who is required to review changes to this file - you'll still be able to PR edits to it!

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+# rainbow fetch 
+src/rainbow-fetch @brunobar79 @skylarbarrera
+
+# raps
+src/raps @brunobar79 @jinchung
+src/raps/registerENS.ts @estebanmino @jxom
+
+# charts, shadow stack, cool modals, sheets
+src/react-native-animated-charts @osdnk @terrysahaidak
+src/react-native-cool-modals @osdnk @terrysahaidak
```
[the remark this answers is not in the document]
Comment (jinchung) [reply CODEOWNERS:103]: cool - we will add bruno instead

**Pull request merged**